src/mon/MDSMonitor.cc

```diff
@@ -1308,7 +1309,7 @@ mds_gid_t MDSMonitor::gid_from_arg(const FSMap &fsmap, const string &arg, ostrea
     // Not a role or a GID, try as a daemon name
     const MDSMap::mds_info_t *mds_info = fsmap.find_by_name(arg);
     if (!mds_info) {
-      ss << "MDS named '" << arg
+      ss << std::endl << "MDS named '" << arg
```

Comment:
nit: I find this style odd

```
Error ENOENT: 
rank/GID '1234' does not exist
rank/GID '55678' does not exist
```

I would personally prefer it to be like this -
```
Error ENOENT: rank/GID '1234' does not exist
Error ENOENT: rank/GID '55678' does not exist
```

Reply:
@rishabh-d-dave is it fine if I hardcode `Error ENOENT:` in  
```
ss << std::endl << "rank/GID '" << arg
	 << "' does not exist";
```

It will be like:
```
ss << std::endl << "Error ENOENT: rank/GID '" << arg
	 << "' does not exist";
```